packages/react-chat/src/components/Modals/WalletModal.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import styled from "styled-components";
+import { WalletConnectIcon } from "../Icons/WalletConnectIcon";
+
+
+import { Modal } from "./Modal";
+import { Heading, Section, Text } from "./ModalStyle";
+// import walletConnect from '../../assets/coinbasewallet.png';
+// import walletConnect from '../../assets/metamask.png';
+
+export const WalletModalName = "WalletModal";
+
+export function WalletModal() {
+
+  return (
+    <Modal name={WalletModalName}>
+      <Section>
+        <Heading>Connect an Ethereum Wallet</Heading>
+      </Section>
+      <MiddleSection>
+        <Text>Choose a way to chat using your Ethereum address.</Text>
+        <Wallets>
+        <Wallet>
+        <Heading>WalletConnect</Heading>
+        <WalletConnectIcon />
+        </Wallet>
+        <Wallet>
+        <Heading>Coinbase Wallet</Heading>
+        <WalletLogo src='../../assets/coinbasewallet.png' alt="coinbase wallet logo" />
```

Comment:
@Szymx95 please check why there is an error if I try to put import  file in img src like this 

`<WalletLogo src={walletConnect} alt="coinbase wallet logo" />`

`ERROR in ../react-chat/dist/esm/src/components/Modals/WalletModal.js 3:0-60
 | Module not found: Error: Can't resolve '../../assets/coinbasewallet.png' in '/Users/maria/Projects/dappconnect-chat-sdk/packages/react-chat/dist/esm/src/components/Modals'`

Reply:
She using `src={walletConnect}` it's evaluated at runtime and asset in specified location doesn't exist when using literal string then it's evaluated at build